_data/projects/CodiMD.yml

```diff
@@ -0,0 +1,10 @@
+name: CodiMD
+desc: Realtime collaborative markdown notes
+site: https://github.com/codimd/server
+tags:
```

Comment:
@ccoenen Hey, isn't there any languages related to this project? Please add if any.

Reply:
I went over the list of common tags and added all that felt appropriate. Thanks for your suggestion!

Reply:
Great!!